{"ray": {"sid": "39cd0e5909c9494f89918818e1561ac1", "uid": null, "qid": "046e09fef28042be9cb595015021eb46", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [046e09fef28042be9cb595015021eb46]\nTraceback (most recent call last):\n  File \"/home/youseef/anaconda3/envs/openfabric/lib/python3.8/site-packages/openfabric_pysdk/app/worker.py\", line 90, in process\n    output = execution_callback_function(data, ray, self.state)\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 68, in execute\n    context_model, qa_model = model_loading()\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 60, in model_loading\n    context_model = ContextModel()\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 23, in __init__\n    self.retriever = RagRetriever.from_pretrained(\nAttributeError: 'RagRetriever' object has no attribute 'to'\n", "created_at": "2023-12-14T01:09:24.559820"}], "status": "FAILED", "created_at": "2023-12-14T01:09:16.212567", "updated_at": "2023-12-14T01:09:24.567180", "finished": true}, "in": {"text": ["hello, openfabric"]}, "out": {}}